packaging/cmake/rdkafka.targets

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can we reuse packaging/nuget/templates/librdkafka.redist.targets?

Reply:
Unfortunately that .targets file assumes the standard nuget directory layout which is not honoured by the cpack nuget generator. I agree. It's not great. But that makes the 2 ways of generating nuget packages fairly disjoint, therefore re-using the .targets file would not make much sense.

Reply:
Makes sense.